### PR TITLE
Fix layout of CSV Import Widget

### PR DESCRIPTION
**Pull request by droidmonkey**
[NOTE]: # ( Describe your changes in detail, why is this change required? )
[NOTE]: # ( Explain large or complex code modifications. )
[NOTE]: # ( If it fixes an open issue, please add "Fixes #XXX" )
The original layout of CSV Import broke when the window was resized.

## Screenshots
[TIP]:  # ( Do not include screenshots of your actual database! )
ORIGINAL:
image

NEW:
image

## Testing strategy
[NOTE]: # ( Please describe in detail how you tested your changes. )
[TIP]:  # ( We expect new code to be covered by unit tests and documented with doc blocks! )

## Type of change
[NOTE]: # ( Please remove all lines which don't apply. )
- ✅ Bug fix (non-breaking change that fixes an issue)

### CODE REVIEW

File: src/gui/csvImport/CsvImportWidget.ui [hunk 1]
```diff
@@ -498,13 +409,7 @@
...
  </customwidgets>
  <tabstops>
-  <tabstop>comboBoxCodec</tabstop>
-  <tabstop>comboBoxTextQualifier</tabstop>
-  <tabstop>comboBoxFieldSeparator</tabstop>
-  <tabstop>comboBoxComment</tabstop>
-  <tabstop>checkBoxFieldNames</tabstop>
   <tabstop>spinBoxSkip</tabstop>
-  <tabstop>checkBoxBackslash</tabstop>
   <tabstop>tableViewFields</tabstop>
```
Comment (phoerious) [inline src/gui/csvImport/CsvImportWidget.ui:413]: Any reason why you removed these?

Comment (droidmonkey) [reply src/gui/csvImport/CsvImportWidget.ui:413]: Go go crappy Qt Designer! They should not have been removed, good catch.

Comment (phoerious) [reply src/gui/csvImport/CsvImportWidget.ui:413]: Did you fix it? I am still seeing the deletions.

**Comment by droidmonkey** [comment]
Done! I took the time to move the column combo boxes into the actual UI file. Found a bug too, so all good now.

**Pull request merged**